src/ex3/commander.js

```diff
@@ -0,0 +1,56 @@
+import { Command } from "commander";
+import { MainCommander } from "./main-commander.js";
+import { ItemManagerCommander } from "./item-manager-commander.js";
+import { PokemonClient } from "./pokemon-client-commander.js";
+
+const itemManagerCommander = new ItemManagerCommander();
+const pokemonClient = new PokemonClient();
+const mainCommander = new MainCommander(itemManagerCommander, pokemonClient);
+
+mainCommander.init();
+
+const program = new Command();
+
+program
+  .name("todos-manager")
+  .description("Add, delete, show and sort todos")
+  .version("1.0.0");
```

Comment:
extract the version from package.json to keep it up to date

Reply:
I've found NODE_NO_WARNINGS=1

Reply:
I found this: https://stackoverflow.com/questions/59738999/does-the-es-modules-implementation-support-importing-json-files
looks like changing the import to require should solve this (CommonJS)
<img width="770" alt="image" src="https://user-images.githubusercontent.com/5465198/173935204-aa6546f3-ec16-4dbe-b3e2-8eb727d41633.png">

Reply:
I do not see the way to use both import and require, and it seems too much to change everything to require because of this warning :)

For now I added the comment: 
// call `node --no-warnings commander.js`

About NODE_NO_WARNINGS=1: it can be called in terminal or set as an environment variable. But we usually do not push .env to Github, so i think that the difference between calling with --no-warnings and setting  NODE_NO_WARNINGS is not significant.